test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
```

Comment:
We should pick a convention and stick with it for consistency. I'm voting for your second suggestion.